config/brakeman.ignore

```diff
@@ -63,21 +63,21 @@
     {
       "warning_type": "SQL Injection",
       "warning_code": 0,
-      "fingerprint": "4a0e88c74e34b17ceb1396c341da9b9d45c4f3f12b28e8b871e7dbb57ddf8b5e",
+      "fingerprint": "36b0cc45521a9bc41933ac9756d5dd14e0e9f902c602dadd0116d9185473a6c5",
       "check_name": "SQL",
       "message": "Possible SQL injection",
-      "file": "app/models/reasons_for_rejection_feature_metrics.rb",
```

Comment:
When this one was ignored, it was a genuine vulnerability and code was added to stop that. Brakeman previously didn't recognise that the change was enough, but apparently now it does!